tests/frame.cc

```diff
@@ -110,7 +110,7 @@ BOOST_AUTO_TEST_CASE(frame) {
 
   Transform3f shift = Transform3f::Random();
   waist.positionInParentFrame(shift);
-  pinocchio->computeForwardKinematics();
+  pinocchio->computeForwardKinematics() JOINT_POSITION;
```

Comment:
This line seems incorrect to me. Does it compile ?

Reply:
Indeed, this is a bad copy-pasting. JOINT_POSITION should be between parenthesis. It turns out this file is not compiled when package `romeo_description` is not available, which is my case.

Reply:
Actually, this file did not even compile before. There is one other compilation error independent from this PR.

Reply:
I fixed this in the last commit.

Reply:
The best unit tests are those that are not compiled since they never fail.